src/ssh/setup.py

```diff
@@ -7,7 +7,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "1.1.6"
+VERSION = "2.0.0"
```

Comment:
Upgrading to the major version means there is a breaking change in this PR, but this PR doesn't seem to have a breaking change, right?

Reply:
May I ask what is the impact of this breaking change on customers? Please note that we generally do not recommend releasing breaking changes on the sprints outside of the breaking change window **(Build and Ignite sprint)**. For more details about breaking change process, please refer to this guideline [breaking_changes](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azcli_docs/breaking_changes)

Reply:
This will have no impact on non-Arc customers.  The Arc usage is a minority of the usage of the extension, and it is known to those customers leveraging the Arc functionality that it is in preview. This is a symptom of not having a mechanism to mark individual functionalities as preview.

If Arc customers do not have this change, then they will be in a state where they have to have connectivity to a machine in order to enable a connectivity feature, this is not an ideal experience and is a blocker for adoption.  This change allows Arc customers to enable the functionality via an API.

There is no impact to how scripts or automation are written for Az CLI.  In my opinion this is a break on the behavior of the Arc enabled server, not the CLI. Given this functionality is in preview on the Arc side, there is no need to wait for a specific date for the change. The version is changed in the CLI to indicate this break, even though it was not a break on the CLI side of the behavior.

@vthiebaut10, feel free to correct me here if I missed something.

Reply:
Got it, thanks for your detailed explanation. If there is a breaking change that has little impact on CLI customers, upgrading to the major version is acceptable. It will not block the merge of this PR. 
However, could you add the description with `[BREAKING CHANGE]` prefix of this change into the `HISTORY.rst`.

Reply:
@zhoxing-ms I added the [BREAKING CHANGE] prefix to the item in History.